### PR TITLE
Added Calculator

### PR DESCRIPTION
**Pull request by Abhay557**
A Calculator Using Javascript For One Page Html Challenge! And Its Made In Only One Page! Sir @Metroxe

### CODE REVIEW

File: entries/Calculator.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Calculator Made By @Abhay557</title>
+</head>
+<style>
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap');
```
Comment (TheTechRobo) [inline entries/Calculator.html:11]: This is a page requisite, which IIRC is not allowed.

File: entries/Calculator.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
```
Comment (TheTechRobo) [inline entries/Calculator.html:7]: Same with this.

File: entries/Calculator.html (hunk 1 above)
Comment (Abhay557) [reply entries/Calculator.html:11]: Sir Let Owner Take decision!

File: entries/Calculator.html (hunk 2 above)
Comment (Abhay557) [reply entries/Calculator.html:7]: Not Be Rush With Your decision

File: entries/Calculator.html (hunk 1 above)
Comment (Metroxe) [reply entries/Calculator.html:11]: @TheTechRobo is correct, you cannot import fonts from an external URL

File: entries/Calculator.html (hunk 2 above)
Comment (Metroxe) [reply entries/Calculator.html:7]: I'm not sure if this even works, but you should remove any import of files. You can embed the style in a `<style>` tag

File: entries/Calculator.html (hunk 1 above)
Comment (Metroxe) [reply entries/Calculator.html:11]: You can actually use the base64 version of the fonts, since that doesn't require a network request.

**Review by Metroxe — CHANGES_REQUESTED** [review]

File: entries/Calculator.html (hunk 1 above)
Comment (Abhay557) [reply entries/Calculator.html]: Ok Sir, I Have Updated The File And
Yes It Can Be Run Without Those Fonts.

Comment (Abhay557) [reply entries/Calculator.html]: @Metroxe Please See Now ! I Mean Review The Changes

File: entries/Calculator.html (hunk 2 above)
Comment (Abhay557) [reply entries/Calculator.html]: Updated Please See Now

**Pull request merged**